tfx/components/transform/executor_test.py

```diff
@@ -191,10 +194,45 @@ def testDoWithDuplicatePreprocessingFn(self):
       self._transform_executor.Do(self._input_dict, self._output_dict,
                                   self._exec_properties)
 
+  def testDoWithCustomSplits(self):
+    self._exec_properties['splits_config'] = json_format.MessageToJson(
+        transform_pb2.SplitsConfig(analyze_splits=['train'],
+                                   transform_splits=['eval']),
+        preserving_proto_field_name=True)
+    self._exec_properties['module_file'] = self._module_file
+    self._transformed_examples.split_names = artifact_utils.encode_split_names(
+        ['eval'])
+    self._output_dict[executor.TRANSFORMED_EXAMPLES_KEY] = [
+        self._transformed_examples]
+
+    self._transform_executor.Do(self._input_dict, self._output_dict,
```

Comment:
Is there a way to test the the contents of these splits is as expected?
We want to make sure that the right data ends up in the right splits.

Reply:
I see, I will follow the test in ExampleGen to check their sizes.

Reply:
Sg, was this done?

Reply:
Yes, I added assertGreater to compare the transformed examples for 'train' and 'eval'.